### ISSUE TITLE
tsv file with value containing double quote in non-quoted field

### ISSUE THREAD

**Issue by kermorgant** [mikefarah/yq#2156]
**Describe the bug**

A tsv file with non quoted value including quotes like `Caramel "Tradition"` makes yq fail with `parse error ...: bare " in non-quoted-field`

Version of yq: 4.44.3
Operating system: linux
Installed via: binary release

**Input tsv**

test.csv:
```tsv
Code    Name
1010    Caramel "Tradition"
```

**Command**
The command you ran:
```
yq -ptsv --tsv-auto-parse=f  -ojson test.csv
```
Error: bad file 'test.csv': parse error on line 2, column 14: bare " in non-quoted-field

**Additional context**
This error originally came from a tsv file downloaded from google spreadsheet using `?format=csv` query param.

**Comment by mikefarah** [comment]
This is an error coming from golang's CSV parsing library (https://pkg.go.dev/encoding/csv) , seems like you need to do:

```
Code	Name
1010	"a ""quoted-field"""
```
To get it to work.

**Comment by kermorgant** [comment]
Indeed, that worked for my test file but unfortunately, I don't have control of the input file in my work environment (it's a direct download from a google spreadsheet). 
There is a `LazyQuote` setting in [https://pkg.go.dev/encoding/csv](https://pkg.go.dev/encoding/csv) that would maybe offer a workaround. Would that be something `yq` could support ?
